src/main/java/com/cmlteam/serv/Serv.java

```diff
@@ -132,13 +90,17 @@ private static Command parseCommandFromArgs(String[] args) {
       throw printHelpAndExit("Provide at least 1 file/folder to serve", options);
     }
 
-    File file = new File(argList.get(0));
-    if (!file.exists()) {
-      throw printHelpAndExit("File/folder doesn't exist", options);
-    }
+    Set<File> files = new HashSet<>();
+    argList.forEach(argument -> {
+      File file = new File(argument);
+      if (!file.exists()) {
+        throw printHelpAndExit("File/folder doesn't exist", options);
+      }
+      files.add(file);
+    });
```

Comment:
1. Could we please substitute these 2 expressions with a single one using `argList.stream().map(...).collect(Collectors.toSet())` or similar?
1. Now if you provide two identical files but in different form like '/aaa/1.pdf' and '/aaa/../aaa/1.pdf' the resulting .tar archive will have two versions of the same file. Could we please adjust the code to only include the same file one time in such case?

Reply:
resolved